### ISSUE TITLE
Flaky Validation Test

### ISSUE THREAD

**Issue by raikoucloud** [sammchardy/python-binance#14]
https://github.com/sammchardy/python-binance/blob/72d3ead77ca7454ae69da1538bacb868e58f920b/binance/validation.py#L47

```
quantity = 2.24
min_trade = 0.01

quantity / min_trade  
# 224.00000000000003
print quantity / min_trade  
# 224.0
print quantity / min_trade - int(quantity / min_trade) > 0.0  
# True
print 224.0 - int(quantity / min_trade) > 0.0  
# False
```

I think that the float division is causing a weird rounding error typical of python. I don't have a nice fix for this yet, but wanted to let you know at least.

**Comment by sammchardy** [comment]
@raikoucloud thanks for the example, I'll look into a better way of validating for this.

If you need to, you can disable local validation on the order method if you need to.

**Comment by sammchardy** [comment]
@raikoucloud I've released v0.3.2 which has a fix for this validation case
